openhands/controller/agent_controller.py

```diff
@@ -164,6 +165,10 @@ def log(self, level: str, message: str, extra: dict | None = None):
         Args:
             message (str): The message to log.
         """
+        # override log level to 'info' if LOG_ALL_EVENTS is set and level is 'debug'
+        if os.getenv('LOG_ALL_EVENTS') in ('true', '1') and level.lower() == 'debug':
```

Comment:
probably this switch should go into the place where we log the event. There might be other debug logs coming through here that the user doesn't need to see

Reply:
Well, currently it's fine, but yes. We are not going to remember for the new ones that there's a trick. 😅